en/docs/administration/configure-custom-domain.md

```diff
@@ -0,0 +1,27 @@
+# Configure Custom Domain
```

Comment:
Should we consider making the title a bit more intuitive?
Suggestion:
"Customize the Developer Portal Domain"

Reply:
done